src/main/java/com/synopsys/integration/phonehome/request/PhoneHomeRequestBodyBuilder.java

```diff
@@ -39,6 +40,30 @@
     private final List<String> artifactModules = new ArrayList<>();
     private final Map<String, String> metaData = new HashMap<>();
 
+    public static PhoneHomeRequestBodyBuilder createForBlackDuck(String integrationRepoName, String registrationId, String blackDuckServerUrl, String integrationVersion, String blackDuckVersion) {
+        return createForProduct(UniquePhoneHomeProduct.BLACK_DUCK, integrationRepoName, registrationId, blackDuckServerUrl, integrationVersion, blackDuckVersion);
+    }
+
+    public static PhoneHomeRequestBodyBuilder createForCoverity(String integrationRepoName, String customerName, String cimServerUrl, String integrationVersion, String cimVersion) {
+        return createForProduct(UniquePhoneHomeProduct.COVERITY, integrationRepoName, customerName, cimServerUrl, integrationVersion, cimVersion);
+    }
+
+    public static PhoneHomeRequestBodyBuilder createForPolaris(String integrationRepoName, String registrationId, String polarisServerUrl, String integrationVersion, String polarisVersion) {

```

Comment:
We may be able to omit polarisVersion here, since we now know Polaris wants to have versionless architecture, and in fact there is no current way to obtain any sort of "version" programmatically from Polaris.

Further, registrationId should be changed to OrganizationName

Reply:
I have changed to organizationName and since I don't know what "may be able to omit" means, I'm leaving it alone. :)